doc/source/support/frequently-asked-questions.rst

```diff
@@ -218,25 +218,25 @@ How can I preview the documentation in real-time?
 How do I build API .rst files automatically for a standard Python package?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can generate ``.rst`` files using the ``sphinx-apidoc`` extension.
+Here is how you can **automate** the process of generating API documentation for a standard Python package located in the ``doc/source/api`` folder.
 
-#. Install ``sphinx-apidoc``:
-
-    .. code-block:: bash
+.. note::
 
-        pip install sphinx-apidoc
+  Your package is considered a **standard package** if it is imported as ``import <package_name>`` instead of ``import <namespace_name>.<package_name>``.
 
-#. Run the following command to generate ``.rst`` under the ``doc/source/api`` directory:
+#. Add ``sphinxcontrib-apidoc`` to the ``requirements/docs.txt`` file.
```

Comment:
``sphinxcontrib-apidoc` is available on conda-forge and I've tested it using a demo package. Works well.